tensorflow/compiler/mlir/tosa/g3doc/legalization.md

```diff
@@ -0,0 +1,4144 @@
+TOSA Lowerings
+==============
+
+Introduction
+============
+
+Overview
+--------
+
+This document provides pseudo-code lowerings from TensorFlow and
+TensorFlow Lite MLIR Dialects
+(https://www.tensorflow.org/mlir/dialects) to an implementation of the
+TOSA Specification (https://developer.mlplatform.org/w/tosa/).
```

Comment:
I'd link to https://mlir.llvm.org/docs/Dialects/TOSA/

Reply:
Link will be fixed in the next patchset.